src/main/java/eu/dissco/core/digitalspecimenprocessor/service/MidsService.java

```diff
@@ -112,13 +120,32 @@ private boolean compliesToMidsTwoBio(DigitalSpecimen attributes) {
         && qualitativeLocationIsValid(attributes)
         && quantitativeLocationIsValid(attributes)
         && (eventIsPresent(attributes) && hasValue(
-        attributes.getOdsHasEvent().get(0).getDwcEventDate(),
-        attributes.getOdsHasEvent().get(0).getDwcVerbatimEventDate(),
-        convertInteger(attributes.getOdsHasEvent().get(0).getDwcYear())))
+        attributes.getOdsHasEvents().get(0).getDwcEventDate(),
+        attributes.getOdsHasEvents().get(0).getDwcVerbatimEventDate(),
+        convertInteger(attributes.getOdsHasEvents().get(0).getDwcYear())))
         && (eventIsPresent(attributes) &&
-        hasValue(attributes.getOdsHasEvent().get(0).getDwcFieldNumber(),
-            attributes.getOdsHasEvent().get(0).getDwcRecordNumber()))
-        && hasValue(attributes.getDwcRecordedBy(), attributes.getDwcRecordedByID());
+        hasValue(attributes.getOdsHasEvents().get(0).getDwcFieldNumber()
+//            ,
+//            attributes.getOdsHasEvents().get(0).getDwcRecordNumber())
```

Comment:
Whoa will fix this, this was waiting on the discussion with Wouter on RecordNumber

Reply:
Removed recordNumber from MIDS calculation as it is not in openDS 0.4.0